### DISCUSSION TITLE
Add codesandbox(es) for the components

### DISCUSSION THREAD

**Discussion by dadamssg** [adobe/react-spectrum#4366] [Ideas]
It would be really nice to have code a codesandbox that shows off the components as they are in the docs with the styling. I think this would make it easier to report bugs and ask questions. We could just fork the sandbox to have reproductions. 

I know there is [this template](https://codesandbox.io/s/react-spectrum-template-syueo) mentioned in the github issue template but it's not a great starting point in my opinion.

I also get an error when trying to use `react-aria-components` for some reason(?). When it does work, i'll have to copy over the styling. I think it would be much better streamlined if there was an official sandbox that includes the examples and their styling. 

![image](https://user-images.githubusercontent.com/881986/231189157-506c1d42-84c7-45a4-ab48-462e4f750e19.png)

**Comment by devongovett** [comment]
Looks like it's pulling in the wrong version. You need to use `^1.0.0-alpha.2` as the version number. `0.0.4` is an old unrelated package, and we took over the name.

**Reply by dadamssg** [reply]
ah. seems that codesandbox isn't an option then. It doesn't give me the option to use `^1.0.0-alpha.2`.

![image](https://user-images.githubusercontent.com/881986/231245680-370e6354-eebe-444c-a057-0081e0b9d4f1.png)

**Reply by snowystinger** [reply]
You can edit the package.json directly
https://codesandbox.io/s/sad-hawking-unmsuf?file=/src/App.js

**Reply by dadamssg** [reply]
ah cool. good tip!
